controllers/networking-calico/charts/internal/calico/templates/psp/calico-typha-horizontal-autoscaler-clusterrole.yaml

```diff
@@ -0,0 +1,15 @@
+
+apiVersion: {{ include "rbacversion" . }}
+kind: ClusterRole
+metadata:
+  name: garden.sapcloud.io:psp:kube-system:typha-cpha
```

Comment:
Can we call all the new resources `gardener.cloud:...` instead of `garden.sapcloud.io:...`?